Mathlib/Topology/Algebra/Category/ProfiniteGrp.lean

```diff
@@ -0,0 +1,171 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Nailin Guan, Yuyang Zhao
+-/
+import Mathlib.Algebra.Category.Grp.FiniteGrp
+import Mathlib.Topology.Category.Profinite.Basic
+import Mathlib.Topology.Algebra.ContinuousMonoidHom
+
+
+/-!
+
+# Category of Profinite Groups
+
+We say `G` is a profinite group if it is a topological group which is compact and totally
+disconnected.
+
+## Main definitions and results
+
+* `ProfiniteGrp` is the type of profinite groups.
+
+* `Pi.profiniteGrp` : The product of profinite groups is also a profinite group.
```

Comment:
The declaration is `pi` (with small `p`), no? And rather than product, I would say "a pi-type".